Alakdan & assembly

Zentrus opened the pull request:
closes #15
closes #13 
- updated assembly guide
- replaced motor
- replaced thermistor specification
- updated Sadulli version
- updated Sadulli readme
- added Alakdan photos

Zentrus committed:
updated assembly

closes #15
- updated assembly guide
- replaced motor
- replaced thermistor specification
- updated Sadulli version
- updated Sadulli readme
- added Alakdan photos

need to update main README table for Alakdan

Zentrus committed:
updated table

pavel-kirienko commented on README.md:
```diff
@@ -64,6 +63,17 @@ The overcurrent protection is configured to trigger at
+
+Original  | Alternatives
+--------- | -----------------
+KTY81/120 | KTY81/110, KTY81/121, KTY81/122, or KTY81/150
```
```suggestion
KTY81/120 | KTY81/110, KTY81/121, KTY81/122, KTY81/150, 102PS1J, 102PS1G
```

pavel-kirienko commented on assembly_instructions/readme.md:
```diff
@@ -1,36 +1,94 @@
+Connectors PCB                | 1     |                                                                        |
+Logic PCB                     | 1     |                                                                        |
+Power stage PCB               | 1     |                                                                        |
+Thermistor                    | 1     | KTY81/120; Alternatives: KTY81/110, KTY81/121, KTY81/122, or KTY81/150 | PTC thermistor with pre-soldered extension wires.
```
```suggestion
Thermistor                    | 1     | NXP KTY81/120, NXP KTY81/110, NXP KTY81/121, NXP KTY81/122, NXP KTY81/150, Littelfuse 102PS1J, or Littelfuse 102PS1G | PTC thermistor with pre-soldered extension wires.
```

pavel-kirienko reviewed (changes requested):
Some things in this list are not directly related to the motor replacement but are still necessary to do.

- [x] Replace UAVCAN with Cyphal and add links to telega.zubax.com
- [x] We need to increase the hardware overcurrent protection resistance to 27k (R1 on the `connectors` board). There is no purpose in providing this information in the README.
- [x] Remove all generated files from the repository. Repositories are meant to contain only source files. Everything that can be generated from the source files goes into documentation package releases.

Zentrus committed:
Update README.md

Co-authored-by: Pavel Kirienko <pavel.kirienko@gmail.com>

Zentrus committed:
Update assembly_instructions/readme.md

Co-authored-by: Pavel Kirienko <pavel.kirienko@gmail.com>

Zentrus committed:
requested changes

- replaced UAVCAN with Cyphal & added link to telega.zubax.com
- removed all generated files

need help with changing R1 to 27k & need to find new part number

pavel-kirienko commented on assembly_instructions/readme.md:
```diff
@@ -1,36 +1,94 @@
+</p>
+
+3. Motor modification. 
+    * Remove the motor stopper.
```
```suggestion
    * Remove the motor retainer ring.
```

pavel-kirienko commented on assembly_instructions/readme.md:
```diff
@@ -1,36 +1,94 @@
+
+5. Housing Attachment. See photos for reference.
+    * Insert the motor phase wires and the thermistor wires through the housing holes. Ensure proper placement.
+    * Get 4 pieces of DIN 7984 4 mm (motor screws). Attach the motor to the top housing. Use threadlock. 
```
```suggestion
    * Get 4 pieces of DIN 7984 4 mm (motor screws). Attach the motor to the top housing. Use threadlock glue.
```

pavel-kirienko reviewed (changes requested):
I left two minor suggestions. Aide from that, the only remaining issue is to update the resistor I mentioned earlier.

Zentrus commented:
@abualkasim-bakeer can you increase the hardware overcurrent protection resistance to 27k (R1 on the connectors board) in eagle?

Zentrus committed:
Update assembly_instructions/readme.md

Co-authored-by: Pavel Kirienko <pavel.kirienko@gmail.com>

Zentrus committed:
Update assembly_instructions/readme.md

Co-authored-by: Pavel Kirienko <pavel.kirienko@gmail.com>

abualkasim-bakeer commented:
The required change is done.

pavel-kirienko commented:
@abualkasim-bakeer please open a pull request against this branch.

![Uploading image.png…]()

abualkasim-bakeer committed:
Change R1 to 27K (#17)

pavel-kirienko committed:
Remove the mention of the hardware over-current protection resistor from the README

pavel-kirienko committed:
Update README.md
